src/utils/queries/index.mjs

```diff
@@ -13,6 +13,13 @@ import {
 } from '../parser/index.mjs';
 import { getRemark } from '../remark.mjs';
 import { transformNodesToString } from '../unist.mjs';
+import {
```

Comment:
I'd rather a prefixed import

Reply:
Cleaner in this situation, these are a bunch on non prefixed constants, doing a wildcard import here better explains where they belong to.